lib/SILOptimizer/Mandatory/DiagnoseUnknownCompileTimeValues.cpp

```diff
@@ -0,0 +1,263 @@
+//===--- DiagnoseUnknownCompileTimeValues.cpp
```

Comment:
Please implement this pass in swift. We shouldn't add new passes in C++.

Reply:
Done!